StRoot/StRHICfUtil/StRHICfPID.h

```diff
@@ -0,0 +1,40 @@
+#ifndef StRHICfPID_hh
+#define StRHICfPID_hh
+
+#include "StRoot/StRHICfUtil/StRHICfFunction.h"
+#include "TMath.h"
+
+class StRHICfPID : public StRHICfFunction
+{
+  public:
+    StRHICfPID();
+    ~StRHICfPID();
+
+    void init();
+    bool calculate();
+
+    void setPlateEnergy(int tower, int layer, float val);
+
+    int getPID(int tower);
+    float getL20(int tower);
+    float getL90(int tower);
+
+  private:
+    float checkStep(int layer);
+    float layerSumEnergy(int tower, int layer);
+    float findRadiationLength(int tower, float ratio);
+    float calculateEquation(int tower, int layer, float sumE);
+
+    int mPID[kRHICfNtower];
+    float mPlateE[kRHICfNtower][kRHICfNplate];
+    float mPlateSumE[kRHICfNtower];
+    float mL20[kRHICfNtower];
+    float mL90[kRHICfNtower];
```

Comment:
Hi Dmitri,

Thank you for your comments. 
This class is a reconstructor for PID and not an IO class. 
RHICf saving classes, such as StRHICfHit and StRHICfPoint, include mostly single-precision (float) variables. 
StRHICfPID also receives input data as mPlateE from the StRHICfHit class using the setPlateEnergy() function. 
Therefore, I didn’t see the need to change it to double precision.

However, I think calculation functions such as calculateEquation() could be changed to double precision. 
Because that function needs to be more precise.
If you think it would be better to change these variables to double precision, I will make the changes along with others.

Thank you